lib/github_processor.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class GithubProcessor
+  def initialize(event_sender)
+    @event_sender = event_sender
+  end
+
+  def process(json_body)
+    event = GithubMessageFactory.build(json_body)
+
+    send_event(
+      name: event.name,
+      params: event.params
+    )
+  end
+
+  private
+
+  def send_event(name:, params: {})
+    puts "Sending '#{name}' event"
+
+    @event_sender.send_event(
+      headers: { 'X-GitHub-Event' => name },
+      params:
```

Comment:
```suggestion
```

Debug leftover or this using the ruby 3.1 hash literal key omission feature?

Reply:
It's using the hash literal feature. I never turned it off in RuboCop.